executor/executor_xapi.go

```diff
@@ -658,3 +662,50 @@ func binopToPBExpr(client kv.Client, expr *ast.BinaryOperationExpr, tn *ast.Tabl
 	}
 	return &tipb.Expr{Tp: tp.Enum(), Children: []*tipb.Expr{leftExpr, rightExpr}}
 }
+
+func likeToPBExpr(client kv.Client, expr *ast.PatternLikeExpr, tn *ast.TableName) *tipb.Expr {
```

Comment:
Please add some comments to indicate that we only support "xx", "%xx", "xx%", "%xx%" patterns.